src/katgpucbf/accel_utils.py

```diff
@@ -0,0 +1,42 @@
+################################################################################
+# Copyright (c) 2022, National Research Foundation (SARAO)
+#
+# Licensed under the BSD 3-Clause License (the "License"); you may not use
+# this file except in compliance with the License. You may obtain a copy
+# of the License at
+#
+#   https://opensource.org/licenses/BSD-3-Clause
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+"""A collection of utility functions for working with katsdpsigproc."""
```

Comment:
I like what's been done from the katgpucbf point of view, but I get the vague feeling that if this is such a common pattern, maybe it should be built directly into katsdpsigproc.accel?

Reply:
Agreed, I'll file a ticket.

Reply:
NGC-833